src/views/catalog/CreateFromInstanceTypes/utils/utils.ts

```diff
@@ -103,7 +115,7 @@ export const generateVM = (
       },
       preference: {
         // inferFromVolume: `${virtualmachineName}-disk`,
```

Comment:
?

Reply:
removed in here https://github.com/kubevirt-ui/kubevirt-plugin/pull/1305